demos/capi/README.md

```diff
@@ -0,0 +1,89 @@
+# C and C++ API inference demo (C/C++)
+
+This demo creates a new docker image openvino/model_server-capi with two example applications that are using c or c++ ovms API.
```

Comment:
```suggestion
This demo creates a new docker image openvino/model_server-capi with two example applications that are using C or C++ OVMS API.
```

Reply:
C and C++ apps which are using OVMS C API?

Reply:
Ok.